src/compiler/rust.rs

```diff
@@ -595,6 +686,7 @@ static ARGS: [ArgInfo<ArgData>; 33] = [
     take_arg!("--out-dir", PathBuf, CanBeSeparated('='), OutDir),
     take_arg!("--pretty", OsString, CanBeSeparated('='), NotCompilation),
     take_arg!("--print", OsString, CanBeSeparated('='), NotCompilation),
+    take_arg!("--remap-path-prefix", OsString, CanBeSeparated('='), TooHard),
```

Comment:
Remapping path prefixes was already a failure case (since it would read incorrect things from the dep file), this addition just makes it graceful.

Reply:
We should probably figure out how to deal with this longer-term but it's probably fine for now. Was it harmful in the non-distributed case, or only in the distributed case?